### PR TITLE
Update debugging.md

### PR DESCRIPTION
**Pull request by LarkAscending**
1. Changing obsolete RenderObject method debugDescribeSettings() to newer RenderParagraph method debugDescribeChildren().

This is a bit of a guess, so please confirm.

2. Changing link to obsolete scheduler/Scheduler/addPersistentFrameCallback to scheduler/FrameCallback.

### CODE REVIEW

File: debugging.md [hunk 1]
```diff
@@ -255,7 +255,7 @@ rendering tree by calling
 As with `debugDumpApp()`, you can call this more or less any time
 except during a layout or paint phase. As a general rule, calling it
 from a [frame
-callback](https://docs.flutter.io/flutter/scheduler/Scheduler/addPersistentFrameCallback.html)
+callback](https://docs.flutter.io/flutter/scheduler/FrameCallback.html)
```
Comment (Hixie) [inline debugging.md]: you want https://docs.flutter.io/flutter/scheduler/SchedulerBinding/addPersistentFrameCallback.html here

File: debugging.md [hunk 2]
```diff
@@ -497,7 +497,7 @@ dirtied because they might be affected by the new dimensions.
 
 If you write your own render objects, you can add information to the
 dump by overriding
-[`debugDescribeSettings()`](https://docs.flutter.io/flutter/rendering/RenderObject/debugDescribeSettings.html).
+[`debugDescribeChildren()`](https://docs.flutter.io/flutter/rendering/RenderParagraph/debugDescribeChildren.html).
```
Comment (Hixie) [inline debugging.md]: debugFillDescription is the new name

File: debugging.md (hunk 1 above)
Comment (LarkAscending) [inline debugging.md]: Done!

**Comment by Hixie** [comment]
Looks right, except I'd leave the `rendering` in the last link rather than change it to `material`. Both work today but `rendering` is more correct and we may get rid of the `material` one.

**Comment by Hixie** [comment]
In any case:
LGTM

**Pull request merged**